website/docs/addon_resolve_admin.md

```diff
@@ -62,49 +92,43 @@ AYON is using its own window widget inside Resolve, for that reason PySide2 has
 
 <TabItem value="win">
 
-paste to any terminal of your choice
 
 ```bash
-%LOCALAPPDATA%\Programs\Python\Python36\python.exe -m pip install PySide2
+cd <path to RESOLVE_PYTHON3_HOME>
+python.exe -m pip install OpenTimelineIO
 ```
 
 </TabItem>
 <TabItem value="linux">
 
-paste to any terminal of your choice
 
 ```bash
-/opt/Python/3.6/bin/python -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python3 -m pip install OpenTimelineIO
 ```
 
 </TabItem>
 <TabItem value="mac">
 
-paste to any terminal of your choice
 
 ```bash
-~/Library/Python/3.6/bin/python -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python3 -m pip install OpenTimelineIO
 ```
 
 </TabItem>
 </Tabs>
 
-<div class="row markdown">
-
-### Set Resolve's Fusion settings for Python 3.6 interpereter
 
-<div class="col col--6 markdown">
+### Debugging if Python is not working in Resolve
 
 
-As it is shown in below picture you have to go to Fusion Tab and then in Fusion menu find Fusion Settings. Go to Fusion/Script and find Default Python Version and switch to Python 3.6
+#### Set Resolve's Fusion settings for Python 3 interpreter
 
-</div>
+In case Resolve is not showing Ayon menu, please have a look into Resolve's console and click to P3 tab. If it is not working then Python is not correctly connected to Resolve. One of the reasons might be that Fusion is not set to use Python 3.6 interpreter.
```

Comment:
```suggestion
In case Resolve is not showing Ayon menu, please have a look into Resolve's console and click to _Py3_ tab. If it is not working then Python is not correctly connected to Resolve. One of the reasons might be that Fusion is not set to use Python 3.6 interpreter.
```
The tab in the console is called Py3 instead of P3.